Bottom: 3fa3a21441913ce290cefcd09c1cf035af0c7a95
Top:    a1a05e9c8d8632698598888247c7f9a1b35f032b
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-31 18:54:10 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/lib/master.c b/lib/master.c
index 9f53440..f4258d2 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1436,7 +1436,6 @@ int dump_map(struct master *master, const char *type, const char *name)
 				instance = source;
 			} else {
 				struct map_source *map;
-				int res;
 
 				map = source->instance;
 				while (map) {
